src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/DataverseUserPage.java

```diff
@@ -161,6 +167,12 @@ public String init() {
             setCurrentUser((AuthenticatedUser) session.getUser());
             userAuthProvider = authenticationService.lookupProvider(currentUser);
             notificationsList = userNotificationService.findByUser(currentUser.getId());
+            final Set<UserNotification.Type> alwaysMuted = UserNotification.Type.tokenizeToSet(BundleUtil.getStringFromBundle("notification.alwaysMuted"));
```

Comment:
Not sure we should create just another configuration mechanism...

Please have a look at the `SettingsWrapper` already present in this class, it wraps the `SettingsServiceBean` for JSF pages.